docs/reference/ml/apis/delete-dfanalytics.asciidoc

```diff
@@ -0,0 +1,52 @@
+[role="xpack"]
+[testenv="platinum"]
+[[delete-dfanalytics]]
+=== Delete {dfanalytics-jobs} API
+[subs="attributes"]
+++++
+<titleabbrev>Delete {dfanalytics-jobs}</titleabbrev>
+++++
+
+beta[]
```

Comment:
DF-Analytics are released as experimental. Not sure what `beta[]` does, but is it the right thing or there's something else for experimental? This comment is valid throughout.

Reply:
I changed the flag to `experimental []` in all cases.